src/pages/docs/creators/javascript/examples/using-app-keys.md

```diff
@@ -37,14 +37,13 @@ node index.mjs
 
 ## Introduction
 
-To use Yagna REST API, your application must identify itself by api-key.
-In our examples we utilize the `try_golem` key, which is automatically created if you have the `YAGNA_AUTOCONF_APPKEY` variable set in the terminal where you start the Yagna service.
+For using the Yagna REST API, your application must identify itself by an api-key. In our examples, we utilize the `try_golem` key. We assign such value to the `YAGNA_AUTOCONF_APPKEY` variable in the terminal **before** we start the Yagna service. Because of that, such a key is automatically created and available for our exercises.
```

Comment:
This should be reverted to the original "To use"
> For using the Yagna REST API

Reply:
> such value

Reword to "This value"


> Because of that, such a key is automatically created and available for our exercises.


Reword to "Because of that, this key is automatically created and available for our exercises."

The word "such" just needs to be gone 😛